java-util/src/main/java/io/druid/java/util/common/parsers/TimestampParser.java

```diff
@@ -47,13 +50,21 @@
         public DateTime apply(String input)
         {
           Preconditions.checkArgument(input != null && !input.isEmpty(), "null timestamp");
+          String[] inputArray = input.split(" ");
```

Comment:
Would it be possible to roll this feature into `createAutoParser()`? If so then I think that would be more clean & robust.

Reply:
I dont think we can move it into createAutoParser(). Abstracted out some of the things and moved it to ParserUtils